bitnami/etcd/templates/cronjob-defrag.yaml

```diff
@@ -37,9 +37,19 @@ spec:
           {{- if .Values.defrag.cronjob.activeDeadlineSeconds }}
           activeDeadlineSeconds: {{ .Values.defrag.cronjob.activeDeadlineSeconds }}
           {{- end }}
+          serviceAccountName: {{ include "etcd.serviceAccountName" . }}
```

Comment:
Typo here? You're adding `serviceAccountName` twice (here and line 52)

Reply:
Yes, it's a typo, I was trying to infer from other templates if it was better to use `{{ include "etcd.serviceAccountName" . }}` or `{{ .Values.disasterRecovery.cronjob.serviceAccountName | quote }}` (as in `disasterRecovery` CronJob template) and forgot to remove one of the two 😅